### PR TITLE
gives bartender a new cartridge with reagent scanner built in

### PR DESCRIPTION
**Pull request by deathrobotpunch**
[QOL] [catering]

## About the PR 
this PR gives bartenders the reagent scanner built in to their PDA device

## Why's this needed? 
This is needed because the bartender deals a lot in reagents and should have a reagent scanner built in to their PDA and QOL is always needed

## Changelog 

```changelog
(u)deathrobotpunch1
(*)Bartender now has a reagent scanner, medical record scanner and a health scanner built in to their PDA!
```

### CODE REVIEW

**Comment by deathrobotpunch** [comment]
why no merge?

**Comment by ZeWaka** [comment]
> why no merge?

saying this will just make people wait longer to review it

**Comment by MintyPhresh** [comment]
I personally like the idea, but not the execution; Tenders should ideally get a cartridge with medical records (for slow metabolism/underage tracking) and reagent scanning. They don't really need the signaler program...

File: code/obj/item/device/pda2/pda2.dm [hunk 1]
```diff
@@ -295,6 +295,7 @@
 		mailgroups = list(MGD_KITCHEN,MGD_PARTY)
 
 	bartender
+		setup_default_cartridge = /obj/item/disk/data/cartridge/toxins
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/pda2.dm:298]: Yeah make a new cartridge

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/item/device/pda2/pda2.dm (hunk 1 above)
Comment (deathrobotpunch) [reply code/obj/item/device/pda2/pda2.dm:298]: ok thank you

**Comment by deathrobotpunch** [comment]
i’m new to coding and I nearly melted my brain trying to figure out how to commit changes in a new file

**Comment by deathrobotpunch** [comment]
zewaka can you review this already?

File: code/obj/item/device/pda2/pda2.dm (hunk 1 above)
Comment (deathrobotpunch) [reply code/obj/item/device/pda2/pda2.dm]: i made the changes

File: code/obj/item/device/pda2/cartridges.dm [hunk 2]
```diff
@@ -239,6 +239,18 @@ TYPEINFO(/obj/item/disk/data/cartridge/syndicate)
 			src.root.add_file( new /datum/computer/file/pda_program/signaler(src))
 			src.read_only = 1
 
+	bartender
+		name = "\improper drinkmaster 1000"
+		desc = "your friend for making the perfect drink"
```
Comment (Tarmunora) [inline code/obj/item/device/pda2/cartridges.dm:244]: Grammar (punctuation, capitalization, etc.)

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/item/device/pda2/cartridges.dm (hunk 2 above)
Comment (deathrobotpunch) [reply code/obj/item/device/pda2/cartridges.dm]: added!

**Pull request merged**